packages/opc-timeline/README.md

```diff
@@ -0,0 +1,176 @@
+```js
+document.querySelector('#timeline').steps = [ 'Scheduled', 'Loaned', 'Extended', 'Completed'];
+```
+
+### For Angular
+- In your app.module include the `CUSTOM_ELEMENTS_SCHEMA` and import the component
+```js
+import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import '@one-platform/opc-timeline';
+
+@NgModule({
+  declarations: [
+    AppComponent,
+  ],
+  imports: [
+    BrowserModule,
+  ],
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
+  providers: [],
+  bootstrap: [AppComponent]
+})
+export class AppModule { }
+```
+- Add component in any component html template
+```html
+<opc-timeline id="timeline" current-step-index="2" variant="compact">
+    <div slot="timeline-details">
+        <h2>Timeline Details</h2>
+        The timeline component details go right here, it uses a slot named <strong>timeline-details</strong>. Happy coding :)
+    </div>
+</opc-timeline>
+```
+
+### For React
+- Import the component in App.js
+```js
+import '@one-platform/opc-timeline';
+```
+
+- Add component in any component html render
+```html
+<opc-timeline id="timeline" current-step-index="2" variant="compact">
+    <div slot="timeline-details">
+        <h2>Timeline Details</h2>
+        The timeline component details go right here, it uses a slot named <strong>timeline-details</strong>. Happy coding :)
+    </div>
+</opc-timeline>
+```
+---
+## Slots
+- There are three optional slots in the opc-timeline as shown below namely
+  - <strong>```timeline-details```</strong>: To set the details for the timeline
+    - Example
+    ```html
+    <opc-timeline>
+      <div slot="timeline-details" style="color: white;">
+        <h2>Timeline Details</h2>
+        The timeline component details go right here, it uses a slot named <strong>timeline-details</strong>. Happy coding :)
+      </div>
+    </opc-timeline>
+    ```
+  - The timeline label slots
+      - <strong>start-label</strong>: The start label on the left side of the timeline 
+      - <strong>end-label</strong>: The end label on the right side of the timeline
+        - Example code
+      ```html
+      <opc-timeline>
+        <span slot="start-label" style="font-weight: 600; color: white;">Loaned on: Apr 28, 2018</span>
+        <span slot="end-label" style="font-weight: 600; color: white;">Expires on: Oct 27, 2018</span>
+      </opc-timeline>
+      ```
+---
+## Attributes
+### steps
+- Input: Array of strings
+  - Example
+  ```js
+  document.querySelector('#timeline').steps = [ 'Scheduled', 'Loaned', 'Extended', 'Completed'];
+  ```
+- The array would be used as steps. 
+- Adding a falsy value would produce an empty step without string
+  - Example of falsy values
+  ```js
+  document.querySelector('#timeline3').steps = [null, undefined, 0];
+  ```
+  - PS: Adding empty string (e.g. - '') would also create a empty step
+
+### current-step-index
+- Input: Number
+- Sets the active state to the given number
+
+### variant
+- Input: The following strings
```

Comment:
remove this line. start from bullets for variant i.e default and compact. add sub-bullet to explain with text.

Reply:
Makes sense and made the changes